styles/style.css

```diff
@@ -66,7 +71,11 @@ p {
 }
 
 .navigation__item {
-  padding: 0.5rem 0;
+  padding: 0.5rem 1rem;
+}
+
+.navigation__item:hover {
+  background-color: #c05326;
```

Comment:
You did the stretch goals to! Nice! Should also have been a var maybe, though this a matter of your own prefrences wher're you're only using it once